content/en/opentelemetry/instrument/api_support/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: OpenTelemetry API Support
+aliases:
+  - /opentelemetry/interoperability/api_support
+  - /opentelemetry/interoperability/otel_api_tracing_interoperability/
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+      tag: 'Documentation'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+      tag: 'Documentation'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+      tag: 'Documentation'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+      tag: 'Blog'
+algolia:
+  tags: ['otel custom instrumentation']
+---
+
+Datadog tracing libraries provide an implementation of the [OpenTelemetry API][1] for instrumenting your code. This means you can maintain vendor-neutral instrumentation of your services, while still taking advantage of Datadog's native implementation, features, and products. 
+
+{{< img src="/opentelemetry/setup/otel-api-dd-sdk.png" alt="Diagram: OpenTelemetry SDK sends data through OTLP protocol to a Collector with Datadog Exporter, which forwards to Datadog's platform." style="width:100%;" >}}
```

Comment:
This diagram makes it feel like the Datadog Agent is the only option here, but you can also use the Collector. I think that's fine for the diagram, but it may be worth clarifying in the text?